lighthouse-core/audits/seo/manual/structured-data.js

```diff
@@ -6,6 +6,16 @@
 'use strict';
 
 const ManualAudit = require('../../manual/manual-audit');
+const i18n = require('../../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Description of a Lighthouse audit that provides detail on the structured data in a page.  Structured data is standardized data on a page that helps a search engine categorize and understand its contents. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

Comment:
Is "structured data" a specific enough term that we should ask that it should be left untranslated? I really have no idea.

We can always leave it to be translated and users can click on the links to learn more.